setup.py

```diff
@@ -20,36 +20,36 @@
         raise RuntimeError("No version number found!")
 
 setup(
-    name='priority',
+    name="priority",
     version=version,
-    description='A pure-Python implementation of the HTTP/2 priority tree',
+    description="A pure-Python implementation of the HTTP/2 priority tree",
     long_description=long_description,
-    long_description_content_type='text/x-rst',
-    author='Cory Benfield',
-    author_email='cory@lukasa.co.uk',
-    url='https://github.com/python-hyper/priority/',
+    long_description_content_type="text/x-rst",
+    author="Cory Benfield",
+    author_email="cory@lukasa.co.uk",
+    url="https://github.com/python-hyper/priority/",
     project_urls={
-        'Documentation': 'https://python-hyper.org/projects/priority/',
-        'Source': 'https://github.com/python-hyper/priority/',
-        'Tracker': 'https://github.com/python-hyper/priority/issues',
-        'Changelog': 'https://github.com/python-hyper/priority/blob/master/HISTORY.rst',
+        "Documentation": "https://python-hyper.org/projects/priority/",
+        "Source": "https://github.com/python-hyper/priority/",
+        "Tracker": "https://github.com/python-hyper/priority/issues",
+        "Changelog": "https://github.com/python-hyper/priority/blob/master/HISTORY.rst",
     },
-    packages=find_packages(where='src'),
-    package_data={'priority': ['py.typed']},
-    package_dir={'': 'src'},
-    python_requires='>=3.6.1',
-    license='MIT License',
+    packages=find_packages(where="src"),
+    package_data={"priority": ["py.typed"]},
+    package_dir={"": "src"},
+    python_requires=">=3.6.1",
```

Comment:
should this also be bumped? if we no longer run CI tests, we should not claim it works on these Python versions.

Reply:
Raise to py38 or py310?